languages/python/exercises/concept/comments/.meta/example.py

```diff
@@ -0,0 +1,20 @@
+# I am single line comment
+"""" 
+This is multi line comment
+Program Build Date: 13-05-2020
+Author:Saurabh
+Team:Developer
+Project: exercism""" 
+
+bank_name = "State Bank of India "  # This line denotes bank name
+
+print(bank_name) # Bank name printed with out comment on line 3
+
+def looper(): # This program iterate from number 1 to 5
+    i=1
```

Comment:
Let's stick to Python style code formatting - i.e. spaces around the `=`, like so: `i = 1`.

A formatter like [Python Black](https://pypi.org/project/black/) would be a helpful place to start.

More context: in Python, the `=` operator is used to pass arguments; in that context, no spaces are used. When it's used to assign values, spaces are used.

```
def do_something(thing, thing2=None):
    print(thing)
    if thing2 is not None:
         print(thing2)
    return thing

# Passing an argument, don't use spaces
do_something("first")
do_something("first", thing2="second")

# Assigning a value, do use spaces
result = do_something("first", thing2="second")
```